packages/shared/src/components/fields/form/FormWrapper.tsx

```diff
@@ -1,43 +1,46 @@
 import React, { ReactElement, ReactNode } from 'react';
 import classNames from 'classnames';
-import { Button, ButtonVariant } from '../../buttons/Button';
+import { Button, ButtonProps, ButtonVariant } from '../../buttons/Button';
 
 interface Copy {
   left?: string;
   right?: string;
 }
 
-interface FormWrapperProps {
+export interface FormWrapperProps {
   children: ReactNode;
   className?: string;
   form: string;
   copy?: Copy;
-  onLeftClick?(): void;
-  onRightClick?(): void;
+  leftButtonProps?: ButtonProps<'button'>;
+  rightButtonProps?: ButtonProps<'button'>;
+  title?: string;
 }
 
 export function FormWrapper({
   children,
   className,
   form,
   copy = {},
-  onLeftClick,
-  onRightClick,
+  leftButtonProps = {},
+  rightButtonProps = {},
+  title,
 }: FormWrapperProps): ReactElement {
   return (
     <div className={classNames('flex flex-col', className)}>
       <div className="flex flex-row justify-between border-b border-theme-divider-tertiary px-4 py-2">
-        <Button variant={ButtonVariant.Tertiary} onClick={onLeftClick}>
+        <Button {...leftButtonProps} variant={ButtonVariant.Tertiary}>
```

Comment:
Doesn't variant become a prop? (for both)

Reply:
It should not change since it is a component based on guidelines for form actions, otherwise, it would look odd if they are mixed up.

Reply:
Ah ok, so we just allow color change right.

Reply:
Yes, most of the time we only change to cabbage when it comes to submit button.